### PR TITLE
ath79-generic: add Meraki MR16

### PR DESCRIPTION
**Pull request by fl0tu**
So far I can see its just working fine.
Is there a checklist to get devices into master?

### CODE REVIEW

**Review by herbetom — CHANGES_REQUESTED** [review]
The Checklist that needs to be filled out is here: https://github.com/freifunk-gluon/gluon/wiki/Device-Integration-checklist

Additional Information regarding device support can be found under the following URLs:
- https://github.com/freifunk-gluon/gluon/wiki/Device-Integration
- https://gluon.readthedocs.io/en/latest/dev/hardware.html

File: targets/ath79-generic [hunk 1]
```diff
@@ -600,3 +600,9 @@ device('ubiquiti-unifi-ap-outdoor+', 'ubnt_unifi-ap-outdoor-plus', {
 })
 
 device('ubiquiti-unifi-ap-pro', 'ubnt_unifi-ap-pro')
+
+-- Meraki
+
+device('meraki_mr16', 'meraki_mr16', {
+        factory =false,
+})
```
Comment (herbetom) [inline targets/ath79-generic:608]: The Vendors should be alphabetically sorted (and within the Vendors sections the devices as well).

**Comment by blocktrron** [comment]
@fl0tu Are you still on this?

**Comment by fl0tu** [comment]
A lot to learn. No time.

**Pull request closed without merging**